staging/src/k8s.io/client-go/tools/cache/thread_safe_store.go

```diff
@@ -71,11 +71,7 @@ type threadSafeMap struct {
 }
 
 func (c *threadSafeMap) Add(key string, obj interface{}) {
-	c.lock.Lock()
-	defer c.lock.Unlock()
-	oldObject := c.items[key]
-	c.items[key] = obj
-	c.updateIndices(oldObject, obj, key)
```

Comment:
why is updateIndices call removed?

Reply:
the code of `Add` is same as `Update`, we can call  `Update` directly in `Add`